pjsip/src/pjsip/sip_transaction.c

```diff
@@ -2182,16 +2182,18 @@ static void send_msg_callback( pjsip_send_state *send_state,
              * since with 503 normally client should try again.
              * See https://github.com/pjsip/pjproject/issues/870
              */
-            if (-sent==PJ_ERESOLVE || -sent==PJLIB_UTIL_EDNS_NXDOMAIN)
+            if (-sent==PJ_ERESOLVE || -sent==PJLIB_UTIL_EDNS_NXDOMAIN) {
                 sc = PJSIP_SC_BAD_GATEWAY;
-            else
+            } else {
                 sc = PJSIP_SC_TSX_TRANSPORT_ERROR;
+            }
```

Comment:
Perhaps put some description (and/or link to this PR) around here for future ref about this new behavior?